package/src/components/CartCheckoutButton/v1/CartCheckoutButton.md

```diff
@@ -0,0 +1,29 @@
+### CartCheckoutButton
+
+#### Overview
+The `CartCheckoutButton` displays a button that allows the user to continue to checkout their cart.
+
+#### A cart checkout button will be rendered when the user needs to continue to the checkout step in the cart
+```jsx
+const Button = require("../../Button/v1").default;
```

Comment:
I don't think you need to require this component, styleguidest automatically imports them so you can just use `Button` w/o the require